naps/cores/cmv12k/s7_phy.py

```diff
@@ -0,0 +1,472 @@
+from nmigen import *
+from nmigen.lib.cdc import PulseSynchronizer
+
+from naps import StatusSignal, ControlSignal, EventReg, Response, driver_method
+from naps.vendor.xilinx_s7 import Mmcm
+from naps.vendor.xilinx_s7.io import IDelayCtrl, _IDelay, _ISerdes
+
+class PokeReg(EventReg, Elaboratable):
```

Comment:
This seems like it is generally useful, so lets move it to `naps.soc.csr_types` and add a small doc string.

Reply:
Done.